src/main/java/com/plaid/client/response/PaymentRecipientGetResponse.java

```diff
@@ -0,0 +1,27 @@
+package com.plaid.client.response;
+
+import com.plaid.client.model.payment.Address;
+import java.util.List;
+
+public final class PaymentRecipientGetResponse extends BaseResponse {
```

Comment:
Same here between this and the `Recipient` class

Reply:
https://github.com/plaid/plaid-java/pull/183#discussion_r369031265